src/Stamp.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { space, fontSize, color } from 'styled-system'
+import theme from './theme'
+
+const getColorCombination = props => {
```

Comment:
Is this unused?

Reply:
@msafari good call, I just removed `getColorCombination`.